provider/iam/token_exchange.go

```diff
@@ -172,6 +172,12 @@ func (tes *tokenExchangeService) newTokenExchangeRequest(logger *zap.Logger) *to
 	}
 }
 
+// UpdateAPIKey ...
+func (tes *tokenExchangeService) UpdateAPIKey(apiKey string, logger *zap.Logger) error {
+	// This method need not be implemented
```

Comment:
When we take the non-service account route, this token_exchange.go is used. Can we implement the method ?

Reply:
Will consider in the upcoming PRs.